networking/multiple_networks/understanding-multiple-networks.adoc

```diff
@@ -12,7 +12,12 @@ By default, OVN-Kubernetes serves as the Container Network Interface (CNI) of an
 
 * *Secondary networks*: Act as secondary, non-default networks for a pod. Secondary networks provide separate interfaces dedicated to specific traffic types or purposes. Only pod traffic that is explicitly configured to use a secondary network is routed through its interface.
 
-However, during cluster installation, {product-title} administrators can configure alternative default secondary pod networks by leveraging the Multus CNI plugin. With Multus, multiple CNI plugins such as ipvlan, macvlan, or Network Attachment Definitions can be used together to serve as secondary networks for pods.
+The following diagram shows a cluster that has an existing default network infrastructure that uses a physical network interface, `eth0`, to connect to two namespaces. Pods or virtual machines (VMs) in one namespace run in isolation from pods or VMs in the other namespace. You can create only one primary network but multiple secondary networks for each namespace.
+
+.Two namespaces that each has one primary UDN and multiple secondary UDNs
```

Comment:
"Namespaces with one or more secondary UDNs"?

Alternatively, "that each have" - I believe number agreement is with namespaces and not on the single primary UDN here?